tools/mkdfs/mkdfs.c

```diff
@@ -257,6 +291,96 @@ uint32_t add_directory(const char * const path)
     return first_entry;
 }
 
+int compare_dfs_entry_hash(const void *a, const void *b)
+{
+    //Sort in hash order
+    dfs_file_t *entry1 = (dfs_file_t *)a;
+    dfs_file_t *entry2 = (dfs_file_t *)b;
+    if(entry1->path_hash > entry2->path_hash) {
+        return 1;
+    } else if(entry1->path_hash < entry2->path_hash) {
+        return -1;
+    } else {
+        return 0;
+    }
```

Comment:
It can be simplified to `entry2->path_hash - entry1->path_hash`

Reply:
Was not able to simplify it into a subtraction of either order.

Reply:
Right, on second thought this would work for signed integers, not unsigned. So I guess the explicit ifs are ok.